data/companies.yml

```diff
@@ -1,9 +1,9 @@
 # TODO: Replace placeholders
 
 providers:
-  - name: "Istio"
-    logo: "/logos/istio.svg"
-    url: "https://istio.io/"
+  - name: "SAP Business Technology Platform, Kyma Runtime"
+    logo: "/logos/sap-btp-kyma-runtime.svg"
```

Comment:
This should be the SAP logo - the goal of the logo wall is to identify the providers that people will be familiar with.

Otherwise, great stuff - thank you!!

Reply:
good point, updated in 7250ac2.